src/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs

```diff
@@ -23,11 +23,27 @@ public abstract partial class DeriveBytes : System.IDisposable
         public abstract byte[] GetBytes(int cb);
         public abstract void Reset();
     }
+    public abstract partial class ECCurve
```

Comment:
This is new public surface area being added?  Has this gone through an API review?